Extensions/CompositeMetadata.md

```diff
@@ -0,0 +1,45 @@
+
+## Metadata Payload
+This metadata type **MUST** only be used in frame types used to initiate streams.  This includes [`REQUEST_FNF`][rf], [`REQUEST_RESPONSE`][rr], [`REQUEST_STREAM`][rs], and [`REQUEST_CHANNEL`][rc].  The [`SETUP` Frame][s] Metadata MIME Type is `message/x.rsocket.composite-metadata`.
+
+[rc]: ../Protocol.md#frame-request-channel
+[rf]: ../Protocol.md#frame-fnf
+[rr]: ../Protocol.md#frame-request-response
+[rs]: ../Protocol.md#frame-request-stream
+[s]:  ../Protocol.md#frame-setup
+
+### Metadata Contents
+```
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |            Version            |
+    +---------------+---------------+-------------------------------+
+    |  MIME Length  |   Metadata Encoding MIME Type                ...
+    +---------------+-------------------------------+---------------+
+    |              Metadata Length                  |
+    +-----------------------------------------------+---------------+
+    |                     Metadata Payload                         ...
+    +---------------+-----------------------------------------------+
+    |  MIME Length  |   Metadata Encoding MIME Type                ...
+    +---------------+-------------------------------+---------------+
+    |              Metadata Length                  |
+    +-----------------------------------------------+---------------+
+    |                     Metadata Payload                         ...
+    +---------------------------------------------------------------+
+                                   ...
+```
+
+* **Version**: (16 bits = max value 65,535) Unsigned 16-bit integer of version number of the composite metadata payload.
+* **Metadata Payload**: Any number of complete metadata payloads.
+  * **MIME Length**: Encoding MIME Type Length in bytes.
+  * **Metadata Encoding MIME Type**: MIME Type for encoding of Data and Metadata. This SHOULD be a US-ASCII string that includes the [Internet media type](https://en.wikipedia.org/wiki/Internet_media_type) specified in [RFC 2045](https://tools.ietf.org/html/rfc2045). Many are registered with [IANA](https://www.iana.org/assignments/media-types/media-types.xhtml) such as [CBOR](https://www.iana.org/assignments/media-types/application/cbor). [Suffix](http://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xml) rules MAY be used for handling layout. For example, `application/x.netflix+cbor` or `application/x.reactivesocket+cbor` or `application/x.netflix+json`. The string MUST NOT be null terminated.
```

Comment:
I think examples like the zipkin types are better for explaining how it will be used.

Reply:
More important useful than consistency with the main spec?  That's my main reasoning for this text.

Reply:
OK, I've taken another crack at this mentioning routing and tracing.  Happy to keep iterating on the wording here.